pkg/ruler/api.go

```diff
@@ -560,3 +551,21 @@ func (a *API) DeleteRuleGroup(w http.ResponseWriter, req *http.Request) {
 
 	respondAccepted(w, logger)
 }
+
+// exportAlert converts AlertStateDesc to Alert. The returned data structure is suitable
+// to be exported by the user-facing API.
+func exportAlert(d *AlertStateDesc) *Alert {
```

Comment:
```suggestion
func alertStateDescToPrometheusAlert(d *AlertStateDesc) *Alert {
```

nit: Seems like a better fit that aligns with other similar functions in Mimir?

Reply:
Done in https://github.com/grafana/mimir/pull/4099/commits/9a6e465447c77bf9810d135eae13040cfbbf891e